appveyor.yml

```diff
@@ -1,3 +1,5 @@
+image: Visual Studio 2015
```

Comment:
Is this necessary?

This should be coordinated with @mayeut, who's been distributing the different tests between CI providers (cfr. #201, for example)

Reply:
Problem is that newer version of AppVeyor image do not have python 2.7 development tool. So most of test will fail. 
without this pin the tests may start failing for no apparent reason.

Reply:
That might actually be good to know if the default version starts failing?
But let's see if @mayeut has any input here.

Reply:
It's in fact already set to `Visual Studio 2015` in master after #204 was merged.
An image specification was needed for windows / linux matrix builds. (as far as I understood the docs: https://www.appveyor.com/docs/appveyor-yml/)

Reply:
Deal with this will be strange. I all test add check of build system and ban python2.7 if condition. Brr.

Reply:
Ah right. But anyway, seems it's already fixed, then. :)